examples/windows/kubernetes-hyperv.json

```diff
@@ -5,37 +5,42 @@
       "orchestratorType": "Kubernetes",
       "orchestratorRelease": "1.15",
       "kubernetesConfig": {
-               "apiServerConfig" : {
-                 "--feature-gates": "HyperVContainer=true"
-               },
-               "kubeletConfig" : {
-                    "--feature-gates": "HyperVContainer=true"
-                }
-          }
+        "networkPlugin": "azure",
+        "containerRuntime": "containerd",
+        "windowsContainerdURL": "https://k8swin.blob.core.windows.net/k8s-windows/containerd/containerplat-aks-test-0.0.8.zip"
+      }
     },
     "masterProfile": {
       "count": 1,
       "dnsPrefix": "hypervtest",
       "vmSize": "Standard_D2_v3"
     },
     "agentPoolProfiles": [
-	    {
-		"name": "windowspool",
-		"count": 2,
-		"vmSize": "Standard_D2_v3",
-		"availabilityProfile": "AvailabilitySet",
-		"osType": "Windows",
-		"osDiskSizeGB": 128,
-		"extensions": [
-		    {
-                        "name": "winrm"
-		    }
-		]
-	    }
+      {
+        "name": "windowspool",
+        "count": 2,
+        "vmSize": "Standard_D4s_v3",
+        "availabilityProfile": "AvailabilitySet",
+        "osType": "Windows",
+        "osDiskSizeGB": 128
+      }
     ],
     "windowsProfile": {
       "adminUsername": "azureuser",
-      "adminPassword": "replacepassword1234$"
+      "adminPassword": "replacepassword1234$",
+      "enableAutomaticUpdates": false,
+      "sshEnabled": true,
+      "windowsPublisher": "MicrosoftWindowsServer",
+      "windowsOffer": "WindowsServer",
+      "windowsSku": "Datacenter-Core-2004-with-Containers-smalldisk",
+      "imageVersion": "latest",
+      "windowsRuntimes": {
+        "default": "process",
+        "hypervRuntimes": [
+          {"buildID": "17763"},
```

Comment:
Should we change buildID to buildNumber?
Most docs including https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-containers/#handling-multiple-windows-versions-in-the-same-cluster and MSFT docs call this the build number 
(and buildID is not sitting well with me)

Reply:
I thought I got that from the powershell command that returns the version but looks like I mixed it up.

Reply:
updated